docs/content/dagster-cloud/deployment/agents/amazon-ecs/configuration-reference.mdx

```diff
@@ -189,4 +203,12 @@ user_code_launcher:
   <li><strong>Default</strong> - 180 (seconds)</li>
   </ul>
   </ReferenceTableItem>
+  <ReferenceTableItem
+  propertyName="config.server_resources">
+  The resources that the agent should allocate to the ECS Service for each code location that it creates. If set, must be a dictionary with a `cpu` and/or `memory` key. Note that [Fargate tasks only support certain combinations of CPU and memory.](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html)
```

Comment:
```suggestion
  The resources that the agent should allocate to the ECS service for each code location that it creates. If set, must be a dictionary with a <code>cpu</code> and/or <code>memory</code> key. <strong>Note</strong>: <a href="https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html">Fargate tasks only support certain combinations of CPU and memory</a>.
```

Unfortunately we can't (currently) use Markdown in these table components. Any formatting has to be HTML.